cdds/cdds/configure/user_config.py

```diff
@@ -152,7 +156,7 @@ def produce_user_configs(request: Request, requested_variables_list: RequestedVa
     return user_configs
 
 
-def get_halo_removal_attributes(request: Request):
+def get_halo_removal_attributes(request: Request, model_id: str = None,):
```

Comment:
docstring and arguments inconsistent, but comment above (about removing model_id argument) would resolve this